packages/themes/src/presets/alternate.css

```diff
@@ -105,7 +109,7 @@
   --default-theme-color-orange: var(--default-theme-color-1);
   --default-theme-color-purple: var(--default-theme-color-1);
 }
-.download-cta a,
+.references-rendered .download-cta .download-button,
 .references-rendered .markdown a {
   color: var(--default-theme-color-1) !important;
```

Comment:
Can we remove these importants with the added specificity?

Reply:
For some reason `markdown a` has an important tags on them. We need to figure out why that is and then we can remove these importants